src/afl-fuzz.c

```diff
@@ -1119,7 +1124,7 @@ int main(int argc, char **argv_orig, char **envp) {
 
   }
 
-  if (afl->queue_cur) show_stats(afl);
+  // if (afl->queue_cur) show_stats(afl);
```

Comment:
intentional? So show_stats() is unnecessary here?

Reply:
yes, removing it for the moment. seems not necessary and costs a bit of speed. but leaving it in place for the moment